src/cf-forms.less

```diff
@@ -448,6 +468,383 @@ textarea {
+        </label>
+        <br>
+        <label for="custom-input_checkbox-ipsum">
+            <input class="custom-input"
+                   id="custom-input_checkbox-ipsum"
+                   type="checkbox"
+                   name="checkboxes-styled"
+                   value="ipsum">
+            Ipsum
+        </label>
+        <br>
+        <label for="custom-input_checkbox-dolor">
+            <input class="custom-input"
+                   id="custom-input_checkbox-dolor"
+                   type="checkbox"
+                   name="checkboxes-styled"
+                   value="dolor">
+            Dolor
+        </label>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          label
+            input[type=checkbox].custom-input
+      notes:
+        - "Use the markup structure above to add .custom-input elements."
+    - name: Custom radio button demo with initialization markup
+      markup: |
+        <label for="custom-input_radio-lorem">
+            <input class="custom-input"
+                   id="custom-input_radio-lorem"
+                   type="radio"
+                   name="radios-styled"
+                   value="lorem">
+            Lorem
+        </label>
+        <br>
+        <label for="custom-input_radio-ipsum">
+            <input class="custom-input"
+                   id="custom-input_radio-ipsum"
+                   type="radio"
+                   name="radios-styled"
+                   value="ipsum">
+            Ipsum
+        </label>
+        <br>
+        <label for="custom-input_radio-dolor">
+            <input class="custom-input"
+                   id="custom-input_radio-dolor"
+                   type="radio"
+                   name="radios-styled"
+                   value="dolor">
+            Dolor
+        </label>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          label
+            input[type=radio].custom-input
+      notes:
+        - "Use the markup structure above to add .custom-input elements."
+    - name: JavaScript setup
+      markup: |
+        <script src="//code.jquery.com/jquery-1.11.0.min.js"></script>
+        <script src="../../static/js/jquery.custom-input.js"></script>
+  tags:
+    - cfgov-forms
+*/
+
+.custom-input {
+
+    @input-width-px: 20px;
+    @input-width: unit(@input-width-px / @base-font-size-px, em);
+
+    // Hide the real input by using absolute positioning to layer a styled div
+    // over it.
+    &_input {
+        position: absolute;
+            top: 4px;
+            left: 4px;
+        z-index: 0;
+        // Use this to try and make the input as small as possible so that it
+        // doesn't stick out from underneath .input_checkbox or .input_radio.
+           -moz-appearance: none;
+        -webkit-appearance: none;
+    }
+
+    // Hide the real input by using absolute positioning to layer a styled div
+    // over it.
```

Comment:
If we used pseudo elems instead of a div we could use the native :hover, :checked, :active instead of doing it w/ JS. Here's a pretty nice example http://codepen.io/Vestride/pen/dABHx